inst/extdata/level-of-service-table-complete.csv

```diff
@@ -14,25 +19,35 @@ Speed Limit (mph),AADT,cycle_segregation,level_of_service
 20 mph,4000+,Painted Cycle Lane,1
 20 mph,4000+,Segregated Track (narrow),2
 20 mph,4000+,Segregated Track (wide),3
-30 mph,0 to 1999,Mixed Traffic Street,2
-30 mph,0 to 1999,Off road cycleway,3
-30 mph,0 to 1999,Painted Cycle Lane,2
-30 mph,0 to 1999,Segregated Track (narrow),2
-30 mph,0 to 1999,Segregated Track (wide),2
-30 mph,2000 to 3999,Mixed Traffic Street,1
+30 mph,0 to 999,Mixed Traffic Street,2
+30 mph,0 to 999,Off road cycleway,3
+30 mph,0 to 999,Painted Cycle Lane,2
+30 mph,0 to 999,Segregated Track (narrow),2
+30 mph,0 to 999,Segregated Track (wide),2
+30 mph,1000 to 1999,Mixed Traffic Street,1
+30 mph,1000 to 1999,Off road cycleway,3
+30 mph,1000 to 1999,Painted Cycle Lane,1
+30 mph,1000 to 1999,Segregated Track (narrow),2
+30 mph,1000 to 1999,Segregated Track (wide),2
+30 mph,2000 to 3999,Mixed Traffic Street,0
 30 mph,2000 to 3999,Off road cycleway,3
 30 mph,2000 to 3999,Painted Cycle Lane,1
-30 mph,2000 to 3999,Segregated Track (narrow),2
+30 mph,2000 to 3999,Segregated Track (narrow),1
 30 mph,2000 to 3999,Segregated Track (wide),2
-30 mph,4000+,Mixed Traffic Street,1
+30 mph,4000+,Mixed Traffic Street,0
```

Comment:
I don't think this is right. Under Matt's new speed classification system, 30 mph max is corresponds to 30-50 kph, in which case this would be 1, right?

![image](https://github.com/user-attachments/assets/6f7a5082-4097-45dc-ab25-c859d7dc4764)